x-pack/plugins/security_solution/public/detections/components/rules/description_step/helpers.tsx

```diff
@@ -151,7 +151,9 @@ export const buildThreatDescription = ({ label, threat }: BuildThreatDescription
                     href={singleThreat.tactic.reference}
                     target="_blank"
                   >
-                    {tactic != null ? tactic.text : ''}
+                    {tactic != null
+                      ? tactic.text
+                      : `${singleThreat.tactic.name} (${singleThreat.tactic.id})`}
```

Comment:
No need to change or anything, but just noting that instead of a ternary you could use optional chaining + nullish coalescing to pair this down a bit if you'd like.

e.g. 
``` ts
{tactic?.text ?? `${singleThreat.tactic.name} (${singleThreat.tactic.id})`}
```